test/gtest-extra-test.cc

```diff
@@ -22,13 +22,15 @@
 
 namespace {
 
+#if FMT_USE_FCNTL
 // This is used to suppress coverity warnings about untrusted values.
 std::string sanitize(const std::string& s) {
   std::string result;
   for (std::string::const_iterator i = s.begin(), end = s.end(); i != end; ++i)
     result.push_back(static_cast<char>(*i & 0xff));
   return result;
 }
+#endif
```

Comment:
This is no longer used and can be removed completely.

Reply:
Even better, I'll nix it and update.